doc/source/getting-started.rst

```diff
@@ -17,12 +17,30 @@ following things:
 * **Docker** to use containers that standardize computing environments
 * **JupyterHub** to manage users and deploy Jupyter notebooks
 
-To get started, we recommend beginning with the first item on our
-:ref:`main page <creating-your-jupyterhub>`. If you'd like to know how to
-expand and customize your jupyterhub setup, check out
-:ref:`extending-jupyterhub`. Finally, for a more extensive description of the
-tools and services that JupyterHub utilizes, see our 
-:ref:`tools` page.
+You already are well on your way to understanding what it means (procedurally)
+to deploy Jupyterhub. 
+
+For more guidance
```

Comment:
I'd call each of these sections something less conversational and more topic-driven / short. So for this, call it something like "Deployment Guide" and then add a sentence about how we've put together a short walkthrough to deploy JupyterHub from scratch.